### ISSUE TITLE
face of loss

### ISSUE THREAD

**Issue by andrewschultz** [andrewschultz/the-problems-compound#60]
In inventory. When you get it, it gives you a bad face, which allows you to face a bad...
doing anything with face of loss = can't scrunch it up.
